src/main/java/org/breedinginsight/api/model/v1/request/query/TraitsQuery.java

```diff
@@ -19,9 +19,117 @@
 
 import io.micronaut.core.annotation.Introspected;
 import lombok.Getter;
+import org.apache.commons.lang3.StringUtils;
+import java.util.ArrayList;
+import java.util.List;
 
 @Getter
 @Introspected
 public class TraitsQuery extends QueryParams {
+    private String name;
+    private String traitDescription;
+    private String entityAttribute;
+    private String mainAbbreviation;
+    private String synonyms;
+    private String level;
+    private String status;
+    private String methodDescription;
+    private String methodClass;
+    private String methodFormula;
+    private String scaleName;
+    private String scaleClass;
+    private String scaleDecimalPlaces;
+    private String scaleLowerLimit;
+    private String scaleUpperLimit;
+    private String scaleCategories;
+    private String createdAt;
+    private String updatedAt;
+    private String createdByUserId;
+    private String createdByUserName;
+    private String updatedByUserId;
+    private String updatedByUserName;
+    private String termType;
+
+    public SearchRequest constructSearchRequest() {
+        List<FilterRequest> filters = new ArrayList<>();
+        if (!StringUtils.isBlank(getName())) {
+            filters.add(constructFilterRequest("name", getName()));
+        }
+        if (!StringUtils.isBlank(getTraitDescription())) {
+            filters.add(constructFilterRequest("traitDescription", getTraitDescription()));
+        }
+        if (!StringUtils.isBlank(getEntityAttribute())) {
+            filters.add(constructFilterRequest("entityAttribute", getEntityAttribute()));
+        }
+        if (!StringUtils.isBlank(getMainAbbreviation())) {
+            filters.add(constructFilterRequest("mainAbbreviation", getMainAbbreviation()));
+        }
+        if (!StringUtils.isBlank(getSynonyms())) {
+            filters.add(constructFilterRequest("synonyms", getSynonyms()));
+        }
+        if (!StringUtils.isBlank(getLevel())) {
+            filters.add(constructFilterRequest("level", getLevel()));
+        }
+        if (!StringUtils.isBlank(getStatus())) {
+            filters.add(constructFilterRequest("status", getStatus()));
+        }
+        if (!StringUtils.isBlank(getMethodDescription())) {
+            filters.add(constructFilterRequest("methodDescription", getMethodDescription()));
+        }
+        if (!StringUtils.isBlank(getMethodClass())) {
+            filters.add(constructFilterRequest("methodClass", getMethodClass()));
+        }
```

Comment:
I think these two fields need to be combined together for filtering purposes because the UI combines them together for display purposes.  When I tried filtering by that column, the filter only applies to the method description.  

Example UI filtering: on the UI the "Method" column shows "test Measurement".  If I type in "test" as my filter, then the ontology term shows correctly, however, if I type "Measurement", the term does not show in the table.